.github/workflows/CodeQL Security Scan.yml

```diff
@@ -24,17 +24,17 @@ jobs:
       ExtensionVersions: ${{ steps.Set_ExtensionVersions.outputs.ExtensionVersions }}
     steps:
     - name: Checkout Ed-Fi-Extensions
-      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```

Comment:
Please be sure to update https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/blob/main/action-allowedlist/approved.json as well. When you do so, please mark all checkout v3 instances as deprecated (see line 6 for example).

Reply:
Sure . Thank you